package.json

```diff
@@ -70,56 +70,56 @@
     ]
   },
   "dependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-transform-runtime": "^7.1.0",
-    "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
     "@babel/register": "^7.0.0",
-    "@babel/runtime": "^7.1.2",
+    "@babel/runtime": "^7.2.0",
     "async": "~0.9.0",
-    "bluebird": "^3.4.7",
+    "bluebird": "^3.5.3",
     "chai": "~4.1.2",
     "chai-as-promised": "^6.0.0",
     "child-process-debug": "0.0.7",
-    "chokidar": "~1.6.0",
-    "chromedriver": "^2.35",
+    "chokidar": "^2.0.4",
+    "chromedriver": "^2.45.0",
     "colors": "1.1.2",
-    "commander": "^2.9.0",
-    "cucumber": "xolvio/cucumber-js#v1.3.0-chimp.6",
+    "commander": "^2.19.0",
+    "cucumber": "^1.3.1",
```

Comment:
hmm.. I'm not sure this will work, we did have to monkey-patch a few things in cucumber to work with the fiberization. have you tried this version with cucumber, or with other test runners? 

Btw - thank you for your work, we are preparing for this release, I'm talking with @samhatoum to coordinate things (I can merge things on the xolvio side but I don't have npm publish access to some of the internal packages :( )

Reply:
@lgandecki @sirphoenix can we leave `cucumber` at `xolvio/cucumber-js#v1.3.0-chimp.6` for now and get the other merged?

Reply:
Would like to get things working with node v10.x

Reply:
xolvio/cucumber-js does not compile with node 10, that's the reason i changed it.

Reply:
What is needed now is my PRs on fiber-utils and xolvio-sync-webdriverio to be merged first, then we can bump the version here.